apps/condo/pages/ticket/[id]/index.tsx

```diff
@@ -482,51 +482,48 @@ const TicketIdPage = () => {
 
     const { refetch: refetchTicket, loading: ticketLoading, obj: ticket, error } = Ticket.useObject({
         where: { id },
-    }, {
-        fetchPolicy: 'network-only',
     })
 
     const userId = get(user, 'id', null)
     const ticketOrganizationId = get(ticket, 'organization.id', null)
 
     const {
         obj: ticketOrganizationEmployee,
-        loading: ticketOrganizationEmployeeLoading,
     } = OrganizationEmployee.useObject({
         where: {
             user: { id: userId },
             organization: { id: ticketOrganizationId },
         },
     })
 
-    const { canEmployeeReadTicket, ticketFilterQueryLoading } = useTicketVisibility()
-    const isEmployeeReadTicket = canEmployeeReadTicket(ticket)
-
-    const loading = userLoading || currentOrganizationLoading || ticketLoading || ticketOrganizationEmployeeLoading || ticketFilterQueryLoading
     const TicketTitleMessage = useMemo(() => getTicketTitleMessage(intl, ticket), [ticket])
 
-    if (loading) {
+    const ticketOrganizationEmployeeOrganizationId = get(ticketOrganizationEmployee, 'organization.id')
+    const currentEmployeeOrganization = get(organization, 'id')
+
+    useEffect(() => {
+        if (
+            ticketOrganizationEmployeeOrganizationId &&
+            ticketOrganizationEmployeeOrganizationId !== currentEmployeeOrganization
+        ) {
+            selectLink(ticketOrganizationEmployee)
+        }
+    }, [ticketOrganizationEmployeeOrganizationId])
```

Comment:
What if I change organization?
```suggestion
    }, [ticketOrganizationEmployeeOrganizationId, currentEmployeeOrganization])
```